components/eam/src/physics/cam/tropopause.F90

```diff
@@ -1046,11 +1046,14 @@ subroutine tropopause_e90(pstate, tropLev, tropP, tropT, tropZ)
     lchnk = pstate%lchnk
     ncol  = pstate%ncol
 
-    ! get index in the physics constituent list
-    call cnst_get_ind('E90', e90_ndx, abrtf=.true.)
-
     ! get index in the chemistry tracer list
-    e90_ndx2 = get_spc_ndx( 'E90' )
+    e90_ndx2 = get_spc_ndx( 'E90' ) !kzm
```

Comment:
@keziming , With this change, e90_ndx now needs to be initialized to 0 or a negative value upfront. Otherwise, in a configuration without E90 constituent (such as V2's F2010), cnst_get would never be called, but a random positive value would lead to the "if (e90_ndx > 0)  then" block starting line